src/EditorFeatures/Core/Extensibility/Navigation/NavigableItemFactory.DeclaredSymbolNavigableItem.cs

```diff
@@ -53,6 +51,17 @@ public DeclaredSymbolNavigableItem(Document document, DeclaredSymbolInfo declare
                     }
                 });
             }
+
+            private ISymbol FindSymbol()
+            {
+                // Cancellation isn't supported when computing the various properties that depend on the symbol, hence
+                // CancellationToken.None.
+                var semanticModel = Document.GetPartialSemanticModelAsync(CancellationToken.None).GetAwaiter().GetResult();
```

Comment:
Note why it's ok o use partial semantics here.  i.e. because we know the symbol is in the document, and we just need a weak symbol just for things like a description string.